src/static/js/app.js

```diff
@@ -3,8 +3,8 @@
    ========================================================================== */
 
 var $ = require( 'jquery' );
-require( 'cf-expandables' );
-require( 'cf-tables' );
+require( 'cf-expandables/src/Expandable' );
+require( 'cf-tables/src/Table' );
 
```

Comment:
If this supercedes #615, should it pull in the code from that PR to initialize the expandables and tables? 

```
var Expandable = require( 'cf-expandables/src/Expandable' );
var Table = require( 'cf-tables/src/Table' );

Expandable.init();
Table.init();
```

Reply:
sure